src/phoenix/server/api/queries.py

```diff
@@ -71,6 +72,35 @@
 
 @strawberry.type
 class Query:
+    @strawberry.field
+    async def users(
+        self,
+        info: Info[Context, None],
+        first: Optional[int] = 50,
+        last: Optional[int] = UNSET,
+        after: Optional[CursorString] = UNSET,
+        before: Optional[CursorString] = UNSET,
+    ) -> Connection[User]:
+        args = ConnectionArgs(
+            first=first,
+            after=after if isinstance(after, CursorString) else None,
+            last=last,
+            before=before if isinstance(before, CursorString) else None,
+        )
+        stmt = select(models.User).order_by(models.User.email)
```

Comment:
Probably need to omit the system user.

Reply:
Yeah let's figure out these nuances as we go. For now I just want to get the plumbing setup.